Update.java

```diff
@@ -0,0 +1,36 @@
+package jdbcConnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class Update {
+   static final String DB_URL = "jdbc:mysql://localhost/javatask";
+   static final String USER = "root";
+   static final String PASS = "Sathish@2000";
+   static final String QUERY = "SELECT id, first, last, age FROM Registration";
+
+   public static void main(String[] args) {
+      // Open a connection
+      try(Connection conn = DriverManager.getConnection(DB_URL, USER, PASS);
+         Statement stmt = conn.createStatement();
+      ) {		      
+         String sql = "UPDATE Registration " +
```

Comment:
Please use proper variable names. For instance, `query` should be used instead of `sql` and `result`/`queryResult` instead of `rs`